### ISSUE TITLE
dimension mismatch

### ISSUE THREAD

**Issue by Hongzunhei** [Chen-Rao/VD-Diff#6]
​Issue Description:​​
I'm working with image data where the original image size is (128, 128). I downsampled it to generate Low-Quality (LQ) data with size (64, 64). However, when trying to concatenate the LQ and GT (Ground Truth) data in my forward function, I encounter a dimension mismatch.
`def forward(self, x, gt):
    gt0 = self.pixel_unshuffle(gt)  # Shape after operation: ?
    x0 = self.pixel_unshuffle(x)    # Shape after operation: ?
    x = torch.cat([x0, gt0], dim=1)  # Dimension mismatch here
    fea = self.E(x).squeeze(-1).squeeze(-1)
    fea1 = self.mlp(fea)
    return fea1`
Problem:​​
After applying pixel_unshuffle to both x (LQ, originally 64x64) and gt (GT, originally 128x128), the dimensions of x0 and gt0 are incompatible for concatenation along dim=1.

​​Questions:​​

    What output shapes should I expect from pixel_unshuffle for inputs of size 128x128 (GT) and 64x64 (LQ)?
    How should I preprocess the LQ data (64x64) to ensure compatibility with the GT data after pixel_unshuffle?
    Are there additional steps (e.g., resizing, channel alignment) required before concatenation?
